src/network/components/ActionBars/SponsorshipActionBar.tsx

```diff
@@ -158,11 +159,11 @@ export const SponsorshipActionBar: FunctionComponent<{
                         },
                         {
                             label: 'Cumulative sponsored',
-                            value: 'NO DATA IN GRAPH',
+                            value: sponsorship.cumulativeSponsoring + ' DATA',
                         },
                         {
                             label: 'Minimum stake',
-                            value: 'NO DATA IN GRAPH',
+                            value: (sponsorship.minimumStake as string) + ' DATA',
```

Comment:
Why `as string`? Can `sponsorship.minimumStake` be undefined? If so, we can end up seeing `undefined DATA`.